step.yml

```diff
@@ -219,6 +219,12 @@ inputs:
         - "true"
         - "false"
       is_required: true
+  - test_case_numbers: ""
+    opts:
+      title: "Test Casd Numbers"
```

Comment:
There's typo `Casd`.
And I think "Test case numbers" is better, as other parameter names.